web/ui/react-app/src/pages/PanelList.tsx

```diff
@@ -181,7 +182,7 @@ class PanelList extends Component<RouteComponentProps & PathPrefixProps, PanelLi
         </Row>
         {panels.map(({ id, options }) => (
           <Panel
-            onExecuteQuery={this.handleQueryHistory}
+            onExecuteQuery={this.onExecuteQuery}
```

Comment:
The convention in React (at least in the docs like https://reactjs.org/docs/handling-events.html) seems to be to call the attribute for handlers `onX()`, but the actual function implementation it points to `handleX()`. So I think we'll want to have `handleExecuteQuery()` here.